elastic/logs/track.json

```diff
@@ -1,5 +1,8 @@
 {% import "rally.helpers" as rally with context %}
 
+{% set p_packages = (packages | default(["apache", "kafka", "mysql", "nginx", "postgresql", "redis", "system"] | tojson)) %}
```

Comment:
IMO this should be only dictated by `integration_ratios`. See `p_integration_ratios` (and usage of `p_integration_ratios.keys()` in the `elastic/security` track

Reply:
Sure, that sounds good.